index.html

```diff
@@ -31,7 +31,139 @@ <h1>I’m Mariane
      <li><a href=""><img src="images/vector2.svg" alt=""></a></li>
     </ul>
    </section>
+   <section>
+    <article class="snapshoot"><img src="images/Snapshoot.png" alt=""></article>
+    <article>
```

Comment:
- [ ] To keep your HTML semantic, you should consider moving this image to the same article as the rest of its description. Typically, `<article>` are identified by a heading (`<h1>-<h6>` elements). You can read more about articles [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article).
- [ ] Keep in mind that you should also apply the changes to the following articles (lines: 58, 81, 104).